pod/main/templates/block/carousel.html

```diff
@@ -0,0 +1,86 @@
+{% load i18n %}
+{% load video_filters static %}
+
+{% if elements.count >= 1 %}
+<div class="pod-inner edito-carousel">
+    <h2>{{ title |capfirst|truncatechars:43 }}</h2>
+    <div id="carousel_{{ container.slug }}_{{ uniq_id }}" class="carousel slide"
+         data-bs-ride="{% if auto_slide %}carousel{% else %}false{% endif %}">
+        {% if elements.count > 1 %}
+			<div class="carousel-indicators">
+				{% for element in elements %}
+				{% if forloop.first %}
+				<button type="button" data-bs-target="#carousel_{{ container.slug }}_{{ uniq_id }}"
+						data-bs-slide-to="{{ forloop.counter0 }}" class="active" aria-current="true"
+						aria-label="Vidéo {{ forloop.counter0 }}"></button>
```

Comment:
Le aria_label doit être une traduction.
Donc `"{% trans 'video' %}"`

Reply:
`				{% for element in elements %}
                {% with video_number=forloop.counter0 %}
				{% if forloop.first %}
				<button type="button" data-bs-target="#carousel_{{ container.slug }}_{{ uniq_id }}"
						data-bs-slide-to="{{ video_number }}" class="active" aria-current="true"
						aria-label="{% blocktrans %}Video {{ video_number }}{% blocktrans %}"></button>
				{% else %}
				<button type="button" data-bs-target="#carousel_{{container.slug}}_{{ uniq_id }}"
						data-bs-slide-to="{{ video_number }}" aria-label="{% blocktrans %}Video {{ video_number }}{% blocktrans %}"></button>
				{% endif %}
                {% endwith %}
				{% endfor %}`

Un truc comme ça ?

Reply:
Ca m'a l'air pas mal ;)
Juste, je remplacerais `aria-label=` par `title=` histoire que plus de monde profite de ce texte ^^

Reply:
ah oui et en anglais `n°` s’écrit `#`, je mettrais donc  `{% blocktrans %}Video #{{ video_number }}{% blocktrans %}`